relion/__init__.py

```diff
@@ -65,7 +64,43 @@ def getEnviron(cls):
 
     @classmethod
     def isVersion2Active(cls):
-        return cls.getActiveVersion().startswith("2.")
+        return cls.getActiveVersion().startswith(V2_1)
+
+    @classmethod
+    def defineBinaries(cls, env):
+        relion_commands = [('./INSTALL.sh -j %d' % env.getProcessors(),
+                            ['relion_build.log',
+                             'bin/relion_refine'])]
+
+        env.addPackage('relion', version='1.4',
+                       tar='relion-1.4.tgz',
+                       commands=relion_commands,
+                       default=False)
```

Comment:
If default=False the "default" option? If so, remove from here, if not, add to the other binaries except for the last one